docs/reference/schemas/resource/manifest/resolve.md

```diff
@@ -0,0 +1,159 @@
+---
+description: JSON schema reference for the 'resolve' property in a DSC Resource manifest
+ms.date:     01/17/2024
+ms.topic:    reference
+title:       DSC Resource manifest resolve property schema reference
+---
+
+# DSC Resource manifest resolve property schema reference
+
+## Synopsis
+
+Indicates how to call the resource to resolve nested instances from an external source.
+
+## Metadata
+
+```yaml
+SchemaDialect: https://json-schema.org/draft/2020-12/schema
+SchemaID:      https://raw.githubusercontent.com/PowerShell/DSC/main/schemas/2024/04/resource/manifest.resolve.json
+Type:          object
+```
+
+## Description
+
+Defines how DSC must call the DSC Resource to resolve an external source to nested DSC Resource
```

Comment:
It's more accurate to say `nested DSC Configuration` instead of `nested DSC Resource instances`

Reply:
When DSC calls `resolve` is the resource expected to return a configuration document or an array of instances to process? I think returning the former might be higher effort for third-party developers of `Import`-kind resources than the latter.

My naive expectation was that the properties I pass to the resource were to resolve an array of instances that DSC itself then invokes.

This makes it sound like the `Import` kind must resolve to a configuration document, which DSC handles by making a (functional) recursive call to `dsc config <operation>`.

Reply:
Currently, DSC will call resolve and immediately apply the operation as if it was a config.